src/pages/policy/input/ParameterEditor.jsx

```diff
@@ -30,6 +31,17 @@ export default function ParameterEditor(props) {
     startDate,
   );
 
+  useEffect(() => {
+    const [parameterStartDate, parameterEndDate] = policy.reform.data[
+      parameterName
+    ]
+      ? Object.keys(policy.reform.data[parameterName])[0].split(".")
```

Comment:
For this code, can you first confirm that `policy.reform.data[parameterName]` is, in fact, an array with a length greater than 0 before attempting to split? This prevents runtime errors if, for some reason (likely a bug somewhere else), `parameterName` exists, but is empty, `undefined`, `null`, or an `Object` or something.